### PR TITLE
zh-trans: add docs/tasks/tools/install-minikube.md

### PR DESCRIPTION
**Pull request by SataQiu**
zh-trans: add docs/tasks/tools/install-minikube.md

### CODE REVIEW

File: content/zh/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## 安装 Hypervisor
+<!--
+## Install a Hypervisor
+-->
+
+如果您还没有安装一种 hypervisor，那么现在就请为您的操作系统安装合适的 hypervisor 程序:
```
Comment (chenrui333) [inline content/zh/docs/tasks/tools/install-minikube.md]:
```suggestion
如果还没有装过 hypervisor，以下是一些不错的选择:
```

File: content/zh/docs/tasks/tools/install-minikube.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+* Linux：[VirtualBox](https://www.virtualbox.org/wiki/Downloads) 或者
+[KVM](http://www.linux-kvm.org/)。
+<!--
+* Linux: [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or
+[KVM](http://www.linux-kvm.org/).
+-->
+
+  {{< note >}}
+
+  Minikube 也支持一个 `-\-vm-driver=none` 选项，该选项在主机而不是 VM 上运行 Kubernetes 组件。
```
Comment (chenrui333) [inline content/zh/docs/tasks/tools/install-minikube.md]:
```suggestion
  Minikube 也支持 `-\-vm-driver=none` 选项，该选项在主机而非 VM 上运行 Kubernetes 组件。
```

File: content/zh/docs/tasks/tools/install-minikube.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+* 请参照[最新发行](https://github.com/kubernetes/minikube/releases)指导安装 Minikube。
+<!--
+* Install Minikube according to the instructions for the [latest release](https://github.com/kubernetes/minikube/releases).
+-->
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* [通过 Minikube 在本地运行 Kubernetes](/docs/getting-started-guides/minikube/)
```
Comment (chenrui333) [inline content/zh/docs/tasks/tools/install-minikube.md]:
```suggestion
* [用 Minikube 在本地运行 Kubernetes](/docs/getting-started-guides/minikube/)
```

**Comment by chenrui333** [comment]
made couple of comments, nice work!

File: content/zh/docs/tasks/tools/install-minikube.md (hunk 2 above)
Comment (SataQiu) [reply content/zh/docs/tasks/tools/install-minikube.md]: 感谢，已更正！

**Pull request merged**